docs/deploy_retrieval_service.md

```diff
@@ -71,25 +71,25 @@ datastore:
 
 ## Deploy to Cloud Run
 
-1. From the root `database-query-extension` directory, deploy the extension
+1. From the root `genai-database-retrieval-app` directory, deploy the retrieval
    service to Cloud Run using the following command:
 
     * For AlloyDB:
 
         ```bash
-        gcloud run deploy extension-service \
-            --source=./extension_service/\
+        gcloud run deploy retrieval-service \
+            --source=./retrieval_service/\
             --no-allow-unauthenticated \
-            --service-account extension-identity \
+            --service-account retrieval-identity \
             --region us-central1 \
         ```
 
         If you are using a VPC network, use the command below:
```

Comment:
nit: isn't the VPC required?

Reply:
Only for AlloyDB

Reply:
In the future we may want to update this since this says you can use it with or without. We should make a note that the without is Cloud SQL